github/PullRequest.py

```diff
@@ -409,25 +409,36 @@ def create_comment(self, body, commit_id, path, position):
         """
         return self.create_review_comment(body, commit_id, path, position)
 
-    def create_review_comment(self, body, commit_id, path, position):
+    def create_review_comment(
+        self, body, commit, path, line, start_line=None, as_suggestion=False
+    ):
         """
         :calls: `POST /repos/{owner}/{repo}/pulls/{number}/comments <https://docs.github.com/en/rest/reference/pulls#review-comments>`_
         :param body: string
-        :param commit_id: :class:`github.Commit.Commit`
+        :param commit: :class:`github.Commit.Commit`
```

Comment:
Why are you changing  this parameter?

Reply:
Because the expected type is a Commit, not a commit id

Reply:
Right, a bit surprising, can you change the param in `create_comment` as well?